pipelines/rj_crm__disparo_template/utils/validators.py

```
@@ -180,6 +182,123 @@ def log_validation_summary(stats: ValidationStats, context: str = ""):
+                dispatch_date="2000-01-01",  # placeholder para validação estrutural
+                campaign_name=str(campaign_name),
+                SubscriberKey=str(row.get("SubscriberKey", "")),
+                telefone=str(row.get("telefone", "")),
+            )
+        except ValidationError as e:
+            for error in e.errors():
+                validation_errors.append(f"Linha {i}: {error.get('loc', ['?'])[0]} - {error.get('msg', '')}")
+
+    if validation_errors:
+        raise ValueError(
+            f"Dados inválidos nas colunas obrigatórias do DataFrame SF:\n"
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)